app/assets/stylesheets/earthworks.css

```diff
@@ -69,11 +69,7 @@
   box-shadow: -4px 4px 4px 0px var(--stanford-fog-light);
 }
 
-.document-counter {
-  display: none;
-}
-
-.documents-list .documentHeader .index_title {
-  margin-left: .75rem;
-  align-self: center;
+.documents-list .document {
+  padding-top: 0px;
+  margin-top: 0px;
 }
```

Comment:
would it work to override or append to `classes` in the component to add `pt-0 mt-0` so we don't need to do it here?

Reply:
I can check but I am pretty sure I would need to pull out a blacklight component to override the classes.

Reply:
Yup. I would need to override the partial https://github.com/projectblacklight/blacklight/blob/a8b5a3d3a2e8243c8a875d40528eae5c106d83bd/app/views/catalog/_document_list.html.erb#L3 to do this.

Reply:
Actually I might be wrong. Nevermind ignore me. Pretty sure this is doable.